src/Umbraco.Web.UI.Client/src/views/content/assigndomain.html

```diff
@@ -32,11 +32,15 @@ <h5 class="umb-pane-title"><localize key="assignDomain_setDomains">Domains</loca
                         <thead>
                             <tr>
                                 <th>
-                                    <localize key="assignDomain_domain"></localize>
+                                    <div id="assignDomainLabel" localize="title" title="@assignDomain_domain">
```

Comment:
Is this the way to do this? It seems a bit "heavy" to me to add a div here which is a block-level implement, maybe a span will do or maybe this can be even more clever? 🤔

Reply:
I agree, however, this seems to be existing code that was just reformatted?

Reply:
@poornimanayar I think you should be able to add the "id" attribute for the `<localize>` directive, which if I remember correctly would then be put on the output element so the `<span>` would get the `id` attribute, so you don't need to wrap it in the `<div>` and make it inline :-)